caching.md

```diff
@@ -0,0 +1,48 @@
+# Caching
+
+There are 3 main types of caching used within Node.js applications. These include:
+* in-process
+* cluster wide
+* http
+
+## Recommended packages
+
+| Caching type  |  Recommended client        |
+|---------------|----------------------------|
+|in process     | [lru-cache][]              |
+|cluster wide   | [ioredis][]                |
+|http           | N/A                        |
```

Comment:
There is some stuff for express: 
https://medium.com/the-node-js-collection/simple-server-side-cache-for-express-js-with-node-js-45ff296ca0f0

Did not try that thx as usually cache is done on proxies rather than the app itself.